### ISSUE TITLE
Error: Not authorized

### ISSUE THREAD

**Issue by jellypuno** [zowe/zowe-on-the-go#2]
This is a known issue and a ticket is opened in api-layer

https://github.com/zowe/api-layer/issues/384

**Comment by jalel01** [comment]
Hi @jellypuno i'm considering the priority of 384 for the API  ML squad and noted that you've marked this as high priority. Yet i see you have a workaround. Is this blocking your development in any way?

**Comment by jellypuno** [comment]
@jalel01 Yes. you cannot use the mobile app unless this is implemented. The workaround that we have is to have a separate API-ML that has the fix but nobody can use it but us. So it would be really great to have this fix implemented in all API-ML instances. :)
